mne/viz/evoked.py

```diff
@@ -208,6 +208,10 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
         raise ValueError('gfp must be boolean or "only". Got %s' % gfp)
 
     scalings = _handle_default('scalings', scalings)
+    if titles is None:
+        append_first_title = ' N$_{ave}$=%d' % (evoked.nave,)
+    else:
+        append_first_title = ''
```

Comment:
`append_first_title = ' N$_{ave}$=%d' % (evoked.nave,) if titles is n None else ''`

I prefer it not only for its brevity, but because it immediately makes clear that `append_first_title` is defined here.

Reply:
Also, isn't `' N$_{ave}$=%d' % evoked.nave'` sufficient here because we know it's only ever an int?